src/com/content-games/games.js

```diff
@@ -166,55 +164,34 @@ export default class ContentGames extends Component {
 //        props.class.push("content-item-boxes");
 
 		var LoadMore = null;
-		const Games = [];
+		var Games = [];
 		const {filter} = props;
 		if ( error ){
 			return <ContentError code="400">"Bad Request : Couldn't load games"</ContentError>;
 		}
 		else if( feed && feed.length > 0 )
 		{
-			feed.forEach( r => {
-				if ( ContentGames.matchesFilter(r.node, filter) ) {
-					Games.push(
-						<ContentItemBox
-							node={r.node}
-							user={props.user}
-							path={props.path}
-							noevent={props.noevent ? props.noevent : null} />
-						);
-				}
-			});
-
-			/*
-				As long as the number of items in the Games array
-				doesn't evenly divide by the number of columns
-				keep adding placeholder elements so that the last
-				row looks nice
-			*/
-			while ( Games.length % layout !== 0 ) {
```

Comment:
Where is this handled downstream? I could not find it and as far as I have seen adding empty placeholders is needed to make the last row items keep the expected size if they are fewer than the number of columns. At least to make it so in all browsers. Should be needed for all flexers really.

Reply:
It's not handled downstream. In my testing it displayed the same in all browsers without placeholders so I removed it.

Reply:
That is fair, but just to be clear we had that low frequency ugly bug with last row sometimes being large, so I wouldn't want it reintroduced. If code can be made simpler though, then that isa strong argument for removing.

Reply:
Yeah, this might need to be undone. I specifically asked @local-minimum to fix said bug.

Reply:
As far as I'm aware it was just a bug because of `flex-grow` being set to 1, whereas in the way the grid is done now it's set to 0. If I can reproduce the bug I'll pop that back in but I really haven't been able to in my testing.

Reply:
Okay, if you can try out the different sizes (1 wide, 2 wide, 3 wide, etc) and everything is equally sized, then it's good.

Reply:
Equally sized in varying sized browser I should say.